integration/xgboost/encryption_plugins/cuda_plugin/src/cuda_utils.h

```diff
@@ -45,7 +46,6 @@ const static unsigned int key_len=1024;
 //const static unsigned int bits=6144;
 //const static unsigned int key_len=3072;
 
-
 const static int TPB=512;
```

Comment:
are these two needed (not sure their meaning)? maybe also remove the unused lines

Reply:
ah I see they are used later in paillier.h, maybe move to there if that's the only place using this?

Reply:
it's also used in cuda_plugin.h, so keep it here for now.